configuration/cardano/shelley-dummy-genesis.json

```diff
@@ -0,0 +1,41 @@
+{
+    "activeSlotsCoeff": 0.99,
+    "protocolMagicId": 838299499,
+    "systemStart": "2020-01-01T00:20:40Z",
+    "genDelegs": {},
+    "updateQuorum": 12,
+    "maxMajorPV": 25446,
+    "maxLovelaceSupply": 100000000,
+    "initialFunds": {},
+    "networkMagic": 403,
```

Comment:
The Byron era _should_ have the same `networkMagic` as the Shelley era. (I know this is not the real Shelley config.)